### PR TITLE
bb-org-overlays: Add PWM overlays

### PR DESCRIPTION
**Pull request by floion**
This adds the device tree overlays needed for using the PWM pins on the beaglebone.

(these overlays are taken from git://github.com/beagleboard/bb.org-overlays.git
at revision b5846ff2b7c903e2d702b8b9bd7106961baee6dd)

Signed-off-by: Florin Sarbu <florin@resin.io>

### CODE REVIEW

File: layers/meta-resin-beaglebone/recipes-kernel/linux/bb-org-overlays/BB-PWM0-00A0.dts [hunk 1]
```diff
@@ -0,0 +1,54 @@
+/*
```
Comment (agherzan) [inline layers/meta-resin-beaglebone/recipes-kernel/linux/bb-org-overlays/BB-PWM0-00A0.dts]: Any reason to copy them and not use them directly from the source code?

Comment (floion) [reply layers/meta-resin-beaglebone/recipes-kernel/linux/bb-org-overlays/BB-PWM0-00A0.dts]: Yes, building the overlays with latest master throws compile errors as the Makefile has been modified in the meanwhile. Quicker to add them like this for the next release

Comment (agherzan) [reply layers/meta-resin-beaglebone/recipes-kernel/linux/bb-org-overlays/BB-PWM0-00A0.dts]: I agree it's quicker but I think it's a hack and brings in duplication that we will have to take care when we update. And in the current implementation, if we update the package and forget to remove these files we will end up shadowing the ones provided by the package.

Comment (floion) [reply layers/meta-resin-beaglebone/recipes-kernel/linux/bb-org-overlays/BB-PWM0-00A0.dts]: I will create an issue and deal with it when I have some time

Comment (floion) [reply layers/meta-resin-beaglebone/recipes-kernel/linux/bb-org-overlays/BB-PWM0-00A0.dts]: So to rephrase, my proposal would be to go with this PR and create an issue to follow-up with fixing the compile error

Comment (floion) [reply layers/meta-resin-beaglebone/recipes-kernel/linux/bb-org-overlays/BB-PWM0-00A0.dts]: I did the Makefile fix and now this PR adds the latest revision of overlays.
Please review

**Pull request merged**